Examples/clim-fig.lisp

```diff
@@ -128,55 +138,68 @@
               (values x1 y1)
               (tracking-pointer (pane)
                 (:pointer-motion (&key window x y)
-                                 (declare (ignore window))
-                                 (set-status-line (format nil "~:(~A~) from (~D,~D) to (~D,~D)"
-                                                          (slot-value *application-frame*
-                                                                      'drawing-mode)
-                                                          (round x1) (round y1)
-                                                          (round x) (round y)))
-                                 (with-output-recording-options (pane :record nil)
-                                   (copy-from-pixmap canvas-pixmap 0 0 pixmap-width pixmap-height pane 0 0)
-                                   (draw-figure pane
-                                                x1 y1 x y
-                                                :fastp t
-						:cp-x1 cp-x1 :cp-y1 cp-y1
-						:cp-x2 cp-x2 :cp-y2 cp-y2)))
+                   (declare (ignore window))
+                   (set-status-line (format nil "~:(~A~) from (~D,~D) to (~D,~D)"
+                                            (slot-value *application-frame*
+                                                        'drawing-mode)
+                                            (round x1) (round y1)
+                                            (round x) (round y)))
+                   (with-output-recording-options (pane :record nil)
+                     (copy-from-pixmap canvas-pixmap 0 0
+                                       pixmap-width pixmap-height pane 0 0)
+                     (draw-figure pane
+                                  x1 y1 x y
+                                  :fastp t
+				  :cp-x1 cp-x1 :cp-y1 cp-y1
+				  :cp-x2 cp-x2 :cp-y2 cp-y2)))
 		(:pointer-button-release (&key event x y)
-                                         (when (= (pointer-event-button event)
-                                                  +pointer-left-button+)
-                                           (return-from processor (values x y))))
+                  (when (= (pointer-event-button event)
+                           +pointer-left-button+)
+                    (return-from processor (values x y))))
                 (:pointer-button-press (&key event x y)
-				       (cond
-					 ((= (pointer-event-button event)
-					     +pointer-right-button+)
-					   (setf cp-x1 x cp-y1 y))
-					 ((= (pointer-event-button event)
-					     +pointer-middle-button+)
-					   (setf cp-x2 x cp-y2 y)))))))
+		  (cond
+		    ((= (pointer-event-button event)
+			+pointer-right-button+)
+		     (setf cp-x1 x cp-y1 y))
+		    ((= (pointer-event-button event)
+			+pointer-middle-button+)
+		     (setf cp-x2 x cp-y2 y)))))))
```

Comment:
(case (ponter-event-button event)
  (+pointer-middle-button+ …)
  …)

Reply:
"delet this!"

I've spoken to an expert on IRC called "jackdaniel" who told me that since `case` compares against symbols, not values, and `(pointer-event-button event)` is `2` and not `(quote +pointer-middle-button+)`, this can't work unless we put the constants into the code. Also, nobody will ever read this. Have a nice day.